### PR TITLE
FEATURE add mock server and main_dev.dart for developement

### PR DESCRIPTION
**Pull request by losiu97**
Created a package with a mock server (for now only authentication and user).
Created main_dev.dart, which runs main.dart + mock server.
Use mock server by using http://localhost:3131 as server URL

### CODE REVIEW

File: packages/mock_server/ios/Flutter/Generated.xcconfig [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// This is a generated file; do not edit or check into version control.
```
Comment (astubenbord) [inline packages/mock_server/ios/Flutter/Generated.xcconfig:1]: Could you please remove the generated files from version control? :)

File: packages/mock_server/lib/mock_server.dart [hunk 2]
```diff
@@ -0,0 +1,161 @@
+library mock_server;
```
Comment (astubenbord) [inline packages/mock_server/lib/mock_server.dart:1]: Not sure about the formatting here, could you try to run `dart format` in the package?

File: packages/mock_server/ios/Flutter/Generated.xcconfig (hunk 1 above)
Comment (losiu97) [reply packages/mock_server/ios/Flutter/Generated.xcconfig]: done

File: packages/mock_server/lib/mock_server.dart (hunk 2 above)
Comment (losiu97) [reply packages/mock_server/lib/mock_server.dart:1]: done

**Pull request closed without merging**